pkg/docker/containers.js

```diff
@@ -17,19 +17,22 @@
  * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
  */
 
-require([
-    "jquery",
-    "base1/cockpit",
-    "docker/mustache",
-    "docker/client",
-    "docker/overview",
-    "docker/details",
-    "docker/image",
-    "docker/storage",
-    "docker/service",
-    "shell/po",
-], function($, cockpit, Mustache, client, overview, container_details, image_details, storage, service, po) {
-    cockpit.locale(po);
+(function() {
+    "use strict";
+
+    var $ = require("jquery");
+    var cockpit = require("cockpit");
+
+    var Mustache = require("mustache");
+    var service = require("service");
+
+    var client = require("./client");
+    var overview = require("./overview");
+    var container_details = require("./details");
+    var image_details = require("./image");
+    var storage = require("./storage.jsx");
+    require("./run");
```

Comment:
Do we need this here? It wasn't in there before and `image.js` already pulls it.